RNA-seq/01-qc_trim_quant.md

```diff
@@ -0,0 +1,244 @@
+
+![per-base-quality-screenshot](https://user-images.githubusercontent.com/19534205/59292316-d6292500-8c4a-11e9-967b-132bd8c54577.png)
+
+Anything below 20, where a phred score of 20 represents a 1 in 100 chance that the call is incorrect, is considered poor quality by FastQC.
+Using `--qualified_quality_phred 15` (which is the default), means scores >= 15 are considered "qualified." 
+Using the default parameters as we do here, reads will be filtered out if >40% of the bases are unqualified.
+You can read more about the quality filter functionality of fastp [here](https://github.com/OpenGene/fastp/blob/master/README.md#quality-filter).
+
+The Salmon documentation notes that, given the way we run `salmon quant`, quantification may be more sensitive to calls that are likely to be erroneous (of low quality) and, therefore, quality trimming may be important.
+Trimming, in contrast to filtering, refers to removing low quality base calls from the (typically 3') end of reads.
+A recent preprint from the Salmon authors ([Srivastava et al. _bioRxiv._ 2019.](https://doi.org/10.1101/657874)) notes that trimming did not effect mapping rates from random publicly available human bulk (paired-end) RNA-seq samples (they used [TrimGalore](https://github.com/FelixKrueger/TrimGalore)). 
+fastp does have [the functionality](https://github.com/OpenGene/fastp#per-read-cutting-by-quality-score) to perform trimming using a sliding window.
+
+_Note that there are two kinds of encoding for phred scores, phred33 and phred64. 
+FastQC guessed that the file for SRR585570 uses Sanger/Illumina 1.9 encoding (phred33). 
+If we had phred64 data, we'd use the `--phred64` flag._
+
+#### `--length_required`
+
+Trimming reads may result in short reads, which may effect gene expression estimates ([Williams et al. _BMC Bioinformatics._ 2016.](https://doi.org/10.1186/s12859-016-0956-2)).
+Using `--length_required 20` means that reads shorter than 20bp will be discarded (similar to what was used in Srivastava et al. above).
+
+#### `--report_title`
+
+When we look at the HTML report, it's helpful to quickly identify what sample the report is for. Using `--report title "SRR585570"` means that the report will be titled "SRR585570" rather than the default ("fastp report").
+
+#### `--json` and `--html`
+
+With these options, we're specifying where the [JSON](https://en.wikipedia.org/wiki/JSON) and HTML reports will be saved (in the `QC/gastric_cancer/fastp_output/` we created) and what the filenames will be. 
+Including the sample name in the filenames again may help us with project organization.
+
+**If we look at** `QC/gastric_cancer/fastp_output/SRR585570_fastp.json` **or the top of the HTML report, we can see that fastp reports certain metrics before and after filtering, which can be very useful in making analysis decisions.**
+
+### Quantification with Salmon
```

Comment:
Because of all the different kinds of headers in the markdown, it might be nice to number the steps here so it's a bit easier for the participant tot navigate and follow along.